integrationTests/testBuiltinFunction.go

```diff
@@ -23,7 +26,23 @@ func (bf *TestBuiltinFunction) IsActive() bool {
 	return true
 }
 
-// IsInterfaceNil --
+// IsInterfaceNil -
 func (bf *TestBuiltinFunction) IsInterfaceNil() bool {
 	return bf == nil
 }
+
+// GenerateOneAddressPerShard -
+func GenerateOneAddressPerShard(shardCoordinator sharding.Coordinator) [][]byte {
+	addresses := make([][]byte, shardCoordinator.NumberOfShards())
+	for i := uint32(0); i < shardCoordinator.NumberOfShards(); i++ {
```

Comment:
It would have been simpler to just force the last byte = shardID, then it should be an address in the respective shard.

Reply:
fixed